tensorflow/lite/delegates/gpu/gl/kernels/add.cc

```diff
@@ -102,6 +99,28 @@ class Add : public NodeShader {
           /*input=*/IOStructure::AUTO,
           /*output=*/IOStructure::AUTO,
       };
+    } else if (hwc_tensor) {
+      *generated_code = {
+        /*parameters=*/{},
```

Comment:
indentation in the {} block needs to be 4, but you indented 2.  so 2 more spaces from L.104 to L.122.

Reply:
The whole file has indentation of 2. So I kept it, if you want to fix indentation use a script.

Reply:
Sorry, we don't follow up with "let the external user submit whatever code they want, and i'll clean it up later with a beautifier."  I suggest you clean it up yourself with `clang-format` with `-style Google`.